### ISSUE TITLE
Error in Slide 89 of Lecture 13

### ISSUE THREAD

**Issue by spendergrsec** [RPISEC/MBE#4]
SMAP became available in Broadwell, not Haswell

**Comment by mcaniw** [comment]
Good catch, looks like Intel pulled it right before release. (It was planned for Haswell)

**Comment by mcaniw** [comment]
Upon further research, it appears that SMAP _was_ introduced with the Haswell chips. It became a feature of _all_ intel chips starting with Broadwell.

**Comment by spender-sandbox** [comment]
Are you sure?  If so, it would be good to provide a source. You seem to be off by one generation -- I have found no evidence of SMAP on Haswell, and in fact not all Broadwell chips have SMAP.  Only all Skylake chips should have it.

-Brad

**Comment by mcaniw** [comment]
[Wikipedia](https://en.wikipedia.org/wiki/Broadwell_(microarchitecture)) seems to indicate that it was indeed introduced in Broadwell. VirtualBox users started to have [issues](https://www.virtualbox.org/ticket/13820) with Broadwell CPUs, seemingly due to SMAP. So it would seem that it was almost definitely introduced with Broadwell.

The confusion comes from the naming scheme that Intel has used. [Wikipedia](https://en.wikipedia.org/wiki/List_of_Intel_CPU_microarchitectures#Roadmap) shows that since Broadwell was a die-shrink of the Haswell microarchitecture, Broadwell CPUs are still sometimes classified as "Haswell". This is the same reasoning that an Ivy Bridge CPU can be classified as "Sandy Bridge era," even though it was technically a generation after.

So I guess that a fully correct statement would be that SMAP became available with the Haswell architecture, starting with Broadwell CPUs.

I know that's a lot of Wikipedia sourcing, but it's the best I can find.
